src/ts/components/buttons/Button.ts

```diff
@@ -24,6 +48,12 @@ export interface ButtonConfig extends ComponentConfig {
    * Default: false
    */
   acceptsTouchWithUiHidden?: boolean;
+
+  /**
+   * The style of the button.
+   * Default: `ButtonStyle.Icon`
```

Comment:
We could directly link the reference

```suggestion
   * Default: {@link ButtonStyle.Icon}
```